<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="OGLWidget" name="glwidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="currentText">
           <string>Kugel</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="maxVisibleItems">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>Kugel</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Würfel</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Zylinder</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopp">
          <property name="text">
           <string>Stopp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetR">
          <property name="text">
           <string>Rotation zurücksetzen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QDial" name="xRot">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDial" name="yRot">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="notchesVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="lbZRot">
        <property name="text">
         <string>Welt_Z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="lbXRot">
        <property name="text">
         <string>Welt_X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="lbYRot">
        <property name="text">
         <string>Welt_Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDial" name="zRot">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Arm 0 - Rotation Z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDial" name="arm0Rotatez">
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Arm 2 - Rotation Z</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Arm 1 - Rotation Z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QDial" name="arm1Rotatez">
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QDial" name="arm2Rotatez">
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>oglwidget.h</header>
   <slots>
    <slot>setParamA(double)</slot>
    <slot>setParamB(double)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
